pkg/featureflag/feature_flag.go

```diff
@@ -13,11 +13,15 @@ import (
 const (
 	InitContainerFeature       = "init-container"
 	ConfidentialComputeFeature = "confidential-compute"
+
+	// Events : support ACI to K8s event translation and broadcasting
+	Events = "events"
```

Comment:
Would you please add a test case for this new feature in the `feature_flag_test.go`?

Reply:
done